invenio_communities/communities/resources/resource.py

```diff
@@ -61,6 +66,28 @@ def create_url_rules(self):
             route("GET", routes["community-requests"], self.search_community_requests),
         ]
 
+    @excluded_communities
```

Comment:
wouldn't it be easier to add excluded_communities in the request_search_args instead of creating the new decorator?

Reply:
Yes, we could, but that decorator is used by other resources that do not need the `excluded_communities`, therefore I thought about creating a dedicated one.